ContactList/ContactList/Validation.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Validation.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/10/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import Foundation
+
+class Validation {
+        
+    static func isValidField(text: String, kindOfField: ValidationFunctions) -> Bool {
+        if text.isEmpty {return true}
+        
+        let textRegEx = kindOfField.getRegExForField
+        
+        let textTest = NSPredicate(format: "SELF MATCHES %@", textRegEx)
+        
+        let result = textTest.evaluate(with: text)
+        
+        return result && (kindOfField == .forTextField ? text.count <= 20 : true)
```

Comment:
I'd recommend to move length limit into appropriate property of `enum ValidationFunctions`